api/message.go

```diff
@@ -129,7 +129,12 @@ func (a *MessageAPI) DeleteMessage(ctx *gin.Context) {
 func (a *MessageAPI) CreateMessage(ctx *gin.Context) {
 	message := model.Message{}
 	if err := ctx.Bind(&message); err == nil {
-		message.ApplicationID = a.DB.GetApplicationByToken(auth.GetTokenID(ctx)).ID
+		application := a.DB.GetApplicationByToken(auth.GetTokenID(ctx))
+		message.ApplicationID = application.ID
+		// if title is empty set application.name as default value
+		if len(message.Title) == 0 {
```

Comment:
Just comparing the strings looks a little cleaner IMO.
```suggestion
		if message.Title == "" {
```

Reply:
May be here need to check strings with whitespaces ? Like this:
```strings.TrimSpace(message.Title) == ""```

Reply:
Yeah good idea.